src/core_modules/capture-core/utils/routing/deriveURLParamsFromLocation.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+
+export const deriveURLParamsFromLocation = (): {| [key: string]: string |} => {
+    const urlSearch = window.location.hash.split('?');
+    return [...new URLSearchParams(urlSearch[1]).entries()].reduce((accParams, entry) => {
```

Comment:
Hey,
I think that in order to have better code readability it would be useful if `urlSearch[1]`, `entry[0]`, `entry[1]` were stored in variables with a meaningful name. WDYT?
Thanks!

Reply:
Yep, should be better now. 
Thanks!